backend-mock/src/utils/middleware.ts

```diff
@@ -9,7 +9,7 @@ export const authenticator = (
 ) => {
 	// Check if bearer token exists
 	const authorization = req.get('authorization')
-	if (!authorization || !authorization.toLowerCase().startsWith('bearer ')) {
```

Comment:
I remember reading somewhere that `bearer` could be in lowercase in the authorization (I might be wrong)
so that is why, I converted it to lowercase and then see if it starts with lowercase `bearer`

Reply:
I usually see it with capital B (e.g. https://jwt.io/introduction/ ), but I guess it doesn't matter so long we agree on it, i'll just do lowercare then.

Reply:
I was converting it to lowercase before checking
so the case doesn't matter now

Reply:
> I was converting it to lowercase before checking so the case doesn't matter now anything should be okay

Ah it's chill then